ports/mimxrt10xx/peripherals/mimxrt10xx/MIMXRT1011/pins.c

```diff
@@ -29,49 +29,97 @@
 #include "py/mphal.h"
 #include "mimxrt10xx/pins.h"
 
-const mcu_pin_obj_t pin_GPIO_00 = PIN(GPIO1, 0, GPIO_00, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_01 = PIN(GPIO1, 1, GPIO_01, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_02 = PIN(GPIO1, 2, GPIO_02, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_03 = PIN(GPIO1, 3, GPIO_03, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_04 = PIN(GPIO1, 4, GPIO_04, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_05 = PIN(GPIO1, 5, GPIO_05, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_06 = PIN(GPIO1, 6, GPIO_06, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_07 = PIN(GPIO1, 7, GPIO_07, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_08 = PIN(GPIO1, 8, GPIO_08, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_09 = PIN(GPIO1, 9, GPIO_09, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_10 = PIN(GPIO1, 10, GPIO_10, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_11 = PIN(GPIO1, 11, GPIO_11, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_12 = PIN(GPIO1, 12, GPIO_12, NO_ADC, 0);
-const mcu_pin_obj_t pin_GPIO_13 = PIN(GPIO1, 13, GPIO_13, NO_ADC, 0);
+const mcu_pin_obj_t pin_GPIO_00 = PIN(GPIO1, 0, 0, GPIO_00, NO_ADC, 0, 0x00000005, 0x000010A0);
```

Comment:
Could we get these magic numbers into defines so it's clearer what the difference between 0x00000005 and 0x00000007 is for example? Probably should use macros like `IOMUXC_SW_PAD_CTL_PAD_SRE`, `IOMUXC_SW_PAD_CTL_PAD_DSE` and so on to get these values in a readable way.

Reply:
There's a good number of different reset values and to be honest I don't know that all of them are actually justified fully by the documentation - they aren't really here to be a value a user ever sets, they're just the default values provided in the datasheet. Minor changes to pullup values to adhere to JTAG/etc specifications, as far as I can tell. If you have specific names or uses for them outside of resetting however I'd be down to implement it.